src/main/java/Storage.java

```diff
@@ -0,0 +1,62 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    String filePath;
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    // load file into TaskList and return it
+    public TaskList getTaskList() throws IOException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        TaskList taskList = new TaskList();
+        while (s.hasNext()) {
+            parseLine(s.nextLine(), taskList);
+        }
+        return taskList;
+    }
+
+    // helper method for getTaskList() to handle each line of file
+    public void parseLine(String taskString, TaskList taskList) {
+        String[] taskDetails = taskString.split(",");
+        for (int i = 0; i < taskDetails.length; i++) {
+            taskDetails[i] = taskDetails[i].trim();
+        }
+        String taskType = taskDetails[0];
+        boolean isTaskDone = taskDetails[1].equals("1");
```

Comment:
Please think of which way is better between using "1" and using any defined value here.